.github/workflows/ci.yml

```diff
@@ -33,55 +33,68 @@ jobs:
           - name: armhf-linux
             host: arm-linux-gnueabihf
             os: ubuntu-20.04
-            packages: g++-arm-linux-gnueabihf
+            packages: g++-arm-linux-gnueabihf qemu-user-static qemu-user
+            dep-opts: "CROSS_COMPILE='yes' SPEED=slow V=1"
+            config-opts: "LIBS='-levent -levent_core -lm' LIBTOOL_APP_LDFLAGS='-all-static' --enable-glibc-back-compat LDFLAGS=-static-libstdc++  --enable-static --disable-shared"
             run-tests: true
             goal: install
           - name: aarch64-linux
             host: aarch64-linux-gnu
             os: ubuntu-20.04
-            packages: g++-aarch64-linux-gnu
+            packages: g++-aarch64-linux-gnu qemu-user-static qemu-user
+            dep-opts: "CROSS_COMPILE='yes' SPEED=slow V=1"
+            config-opts: "LIBS='-levent -levent_core -levent_pthreads -lm' LIBTOOL_APP_LDFLAGS='-all-static' --enable-glibc-back-compat LDFLAGS=-static-libstdc++ --enable-static --disable-shared"
             run-tests: true
             goal: install
           - name: x86_64-linux-dbg
-            host: x86_64-unknown-linux-gnu
+            host: x86_64-pc-linux-gnu
             os: ubuntu-20.04
             run-tests: true
-            dep-opts: "DEBUG=1"
-            config-opts: "CPPFLAGS=-DDEBUG_LOCKORDER"
+            packages: python3-dev python3-dbg python
+            dep-opts: "DEBUG=1 SPEED=slow V=1"
+            config-opts: "LIBTOOL_APP_LDFLAGS='-all-static' LD_LIBRARY_PATH='`pwd`/depends/x86_64-pc-linux-gnu/lib/' CFLAGS='-I`pwd`/depends/x86_64-pc-linux-gnu/include/ -fPIC' LDFLAGS='-L`pwd`/depends/x86_64-pc-linux-gnu/lib -static-libgcc -static-libstdc++' PKG_CONFIG_PATH=`pwd`/depends/x86_64-pc-linux-gnu/lib/pkgconfig LIBS='-levent -levent_core -levent_pthreads -lm' --enable-static"
```

Comment:
Related to previous comment, consider moving the math library (-lm) config options to configure.ac to ensure tests build outside ci.